output/paper/TemplateTex.log

```diff
@@ -0,0 +1,393 @@
+This is pdfTeX, Version 3.141592653-2.6-1.40.22 (TeX Live 2021) (preloaded format=pdflatex 2022.2.4)  31 MAR 2022 11:37
```

Comment:
Maybe create a local .gitignore that ignores TemplateTex.log? (Since we don't want to ignore .LOG files universally.)

Reply:
@jmshapir The problem here is that in /output/paper there is both TemplateTex.log and sconscript.log, so adding a local .gitignore for '*.log' would also ignore our sconscript log file. We could just have the .gitignore ignore specifically 'TemplateTex.log' but this really wouldn't do anything for a future user of the template since they will certainly change the name of the output files. If we would still prefer to do this for the cleanliness of the Template repo, we can.

Reply:
@rcalvo12 can we add a local ignore for .LOG but exempt sconscript.log?

Reply:
@jmshapir apologies, on my first search I didn't think it was possible, but I believe the change in 63dda7c should do the trick.

Reply:
@rcalvo12 I agree, that looks good, thanks!